overthebox/files/etc/udhcpc.user

```diff
@@ -85,7 +85,7 @@ manage_if0() {
 		set dhcp.if${_id}_gw=tag
 		add_list dhcp.if${_id}_gw.dhcp_option=1,$subnet
 		add_list dhcp.if${_id}_gw.dhcp_option=3,$router
-		add_list dhcp.if${_id}_gw.dhcp_option=6,$dns
+		add_list dhcp.if${_id}_gw.dhcp_option=6,$(printf '%s' "$dns" | tr ' ' ',')
```

Comment:
maybe `${dns// /,}`

Reply:
That was my first thought, but shellcheck wasn't happy about it : 
```In POSIX sh, string replacement is undefined. [SC2039]```

Reply:
with `# shellcheck disable=SC2154,SC1091,SC2039` :)